README.md

```diff
@@ -158,8 +158,19 @@ This is experimental.
 ### Prometheus code base
 
 In order to comply with [go mod](https://go.dev/ref/mod#versions) rules,
-Prometheus release number do not exactly match Go module releases. For the
-Prometheus v2.y.z releases, we are publishing equivalent v0.y.z tags.
+Prometheus release number do not exactly match Go module releases.
+
+For the
+Prometheus v3.y.z releases, we are publishing equivalent v0.3y.z tags.
+
+Therefore, a user that would want to use Prometheus v3.00.0 as a library could do:
```

Comment:
Just that it's called v3.0.0, not v3.00.0. I'm afraid we have to explain the leading zero thing for single digit minor releases. (It wasn't necessary for v2 because we only introduced this versioning method when the v2 minor release count was already two digit.)

Reply:
Updated in https://github.com/prometheus/prometheus/pull/15361/commits/9f4f08d0d50f6322c947646b271d8d1a5eb56807